src/freenet/clients/http/N2NTMToadlet.java

```diff
@@ -285,6 +289,14 @@ public static void createN2NTMSendForm(HTMLNode pageNode, boolean advancedMode,
 		}
 		messageForm.addChild("textarea", new String[] { "id", "name", "rows",
 				"cols" }, new String[] { "n2ntmtext", "message", "8", "74" });
+		if (fProxyJavascriptEnabled) {
+			HTMLNode letterCountInfo = messageForm.addChild("div", new String[] {"class"}, new String[] {"n2ntmcountouter"});
+			letterCountInfo.addChild("#", NodeL10n.getBase().getString("N2NTMToadlet.remainingLetters"));
+			letterCountInfo.addChild("span", new String[] {"id"}, new String[] {"n2ntmcount"});
+			letterCountInfo.addChild("script", new String[] {"type"}, new String[] {"text/javascript"})
```

Comment:
This should be done in `countletters.js`, on the document’s “onload” event (or jQuery’s “ready” if we have jQuery available).